guides/guide-for-adding-windows-node.md

```diff
@@ -1,229 +1,402 @@
+kubectl apply -f https://github.com/kubernetes-sigs/sig-windows-tools/releases/latest/download/flannel-overlay.yml
+```
 
-Let's apply the Flannel configuration:
+>  **Note** If you are using another version of kubernetes on your windows node, change v1.24.3 with your own version .
+> To find your version of kubernetes run the following command:
+> `kubeadm version`
+
+5. Apply kube-flannel-rbac.yml from sig-windows-tools/kubeadm/flannel
+Next you will need to apply the configuration that allows flannel to spawn pods and keep them running:
 
 ```bash
+git clone https://github.com/kubernetes-sigs/sig-windows-tools
+kubectl apply -f sig-windows-tools/kubeadm/flannel/kube-flannel-rbac.yml
+```
 
-kubectl apply -f kube-flannel.yml
 
+### Joining a Windows worker node
+
+>  **Note:** All code snippets in Windows sections are to be run in a PowerShell environment with elevated permissions (Administrator) on the Windows worker node.
+
+1. Install ContainerD, wins, kubelet, and kubeadm.
+
+```PowerShell
+git clone https://github.com/kubernetes-sigs/sig-windows-tools
+cd .\sig-windows-tools\kubeadm\scripts\
+
+# Install ContainerD
+.\Install-Containerd.ps1
+
+# Install wins, kubelet and kubeadm
+.\PrepareNode.ps1 -KubernetesVersion v1.24.3 -ContainerRuntime containerD
 ```
 
-After a few minutes, you should see all the pods as running if the Flannel pod network was deployed.
+>  **Note** If you want to install another version of kubernetes, modify v1.24.3 with the version you want to install
+
+
+2. Run `kubeadm` to join the node
+>  **Note** Before joining the node, copy the file from /run/flannel/subnet.env to your windows machine to C:\run\flannel\subnet.env
+>  You will need to create the folders for it
+
+Use the command that was given to you when you ran `kubeadm init` on a control plane host. If you no longer have this command, or the token has expired, you can run `kubeadm token create --print-join-command` (on a control plane host) to generate a new token and join command.
+
+>  **Note:** Do not forget to add `--cri-socket "npipe:////./pipe/containerd-containerd" --v=5` at the end of the join command, if you use ContainerD
+
+3. Install kubectl for windows (optional)
+
+For more information about it : https://kubernetes.io/docs/tasks/tools/install-kubectl-windows/
+
+
+#### Verifying your installation
+
+You should now be able to view the Windows node in your cluster by running:
 
 ```bash
+kubectl get nodes -o wide
+```
 
-kubectl get pods -n kube-system
+If your new node is in the `NotReady` state it is likely because the flannel image is still downloading. You can check the progress as before by checking on the flannel pods in the `kube-system` namespace:
 
+```shell
+kubectl -n kube-system get pods -l app=flannel
 ```
 
-The output should include the Linux flannel DaemonSet as running:
+Once the flannel Pod is running, your node should enter the `Ready` state and then be available to handle workloads.
+
+
+## Configuring Calico hostprocess
```

Comment:
Add note about required Kubernetes/Windows version for hostprocess containers to be available